files/build_templates/sonic_debian_extension.j2

```diff
@@ -439,6 +438,10 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT docker pull calico/
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT docker pull calico/pod2daemon-flexvol:v${K8s_CNI_CALICO_VERSION}
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT docker pull calico/cni:v${K8s_CNI_CALICO_VERSION}
 echo "docker images pull complete"
+sudo cp $IMAGE_CONFIGS/kubernetes/scripts/kube_join.sh $FILESYSTEM_ROOT/usr/bin/
+sudo cp $IMAGE_CONFIGS/kubernetes/scripts/kube_reset.sh $FILESYSTEM_ROOT/usr/bin/
+sudo chmod 755 $FILESYSTEM_ROOT/usr/bin/kube_join.sh
+sudo chmod 755 $FILESYSTEM_ROOT/usr/bin/kube_reset.sh
```

Comment:
Move them to a wheel, so no need to manually install here.

Reply:
Prefer to keep it along with hostcfgd.

Reply:
Good point, move them together with hostcfgd to another repo. No need to change bash if you don't want to.

---
In reply to: [414882038](https://github.com/Azure/sonic-buildimage/pull/4474#discussion_r414882038) [](ancestors = 414882038)

Reply:
I am not sure about moving hostcfgd out of sonic-buildimage repo. If we decide to do, I prefer do it part of another PR - meant for house keeping.